ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -154,7 +154,7 @@ def negative_binomial(n, p, size=None):
 
 @to_ivy_arrays_and_back
 @from_zero_dim_arrays_to_scalar
-def laplace_sample(mean=0.0, scale=1.0, size=None):
+def laplace(mean=0.0, scale=1.0, size=None):
     ret = ivy.exp(
         ivy.random.laplace(mean=mean, scale=scale, size=size, dtype="float64")
```

Comment:
I think ivy doesn't have a `laplace` function(just re-confirm it). In that case, you will have to use ivy compositions to implement this here.

Reply:
The docs should help you with this:
https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends.html#short-frontend-implementations
https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends.html#temporary-compositions

Reply:
Hi @bipinKrishnan . So from my understanding, I should implement the laplace function as per the example in the temporary compositions, right?

Reply:
yes

Reply:
@bipinKrishnan Please review the changes, thanks!